scripts/build_staticdocs.R

```diff
@@ -1,37 +1,26 @@
 #!/usr/bin/env Rscript
 
 library(devtools)
-library(staticdocs)
+library(pkgdown)
+library(magrittr)
 
-pkgs <- list( "db", "settings", "utils","visualization",
-              "modules/allometry","modules/assim.batch",
-              "modules/assim.sequential",
-              "modules/data.atmosphere","modules/data.land",
-              "modules/data.remote","modules/emulator",
-              "modules/meta.analysis","modules/photosynthesis",
-              "modules/priors","modules/rtm",
-              "modules/uncertainty", 
-              "models/biocro", "models/clm45",
-              "models/dalec",
-              "models/ed", "models/gday",
-              "models/linkages","models/lpjguess",
-              "models/maat","models/maespa",
-              "models/preles",
-              "models/sipnet",
-              "all")
- 
-lapply(pkgs, function(x) dir.create(file.path(x, "inst/staticdocs"), recursive = TRUE))
-build.doc <- function(x){
-  setwd(file.path("~/pecan", x))
-  y <- basename(x)
-  build_site(file.path("../", y))
-}
-start = 1
-for(i in start:length(pkgs)){
-  print(i)
-  build.doc(pkgs[[i]])
+pkg_list<- list.dirs(path = ".", full.names = FALSE, recursive = TRUE) %>%
+           grep("man$",., value = TRUE) %>%
+           dirname(.)
+pkg_list <- pkg_list[-1]
+
+lapply(pkg_list, function(x){
+  print(x)
+  tryCatch(
+    {
+      pkgdown::build_site(x,examples = FALSE)
```

Comment:
I would add the arugment `override = list(destination = file.path("docs", x))` or something to put the HTML files in something like `docs/base/utils/`, `docs/models/ed`, etc.